packages/easy-coding-standard/config/config.php

```diff
@@ -37,24 +35,23 @@
         $cacheDirectory .= '_' . StaticVersionResolver::PACKAGE_VERSION;
     }
 
-    // @todo turn these into methods :)
-
-    $parameters->set(Option::CACHE_DIRECTORY, $cacheDirectory);
+    $ecsConfig->cacheDirectory($cacheDirectory);
 
     $cacheNamespace = str_replace(DIRECTORY_SEPARATOR, '_', getcwd());
-    $parameters->set(Option::CACHE_NAMESPACE, $cacheNamespace);
+    $ecsConfig->cacheNamespace($cacheNamespace);
 
     // parallel
     $ecsConfig->parallel();
 
     // how many files are processed in single process
-    $parameters->set(Option::PARALLEL_JOB_SIZE, 60);
-    $parameters->set(Option::PARALLEL_MAX_NUMBER_OF_PROCESSES, 16);
-    $parameters->set(Option::PARALLEL_TIMEOUT_IN_SECONDS, 120);
+    $ecsConfig->parallelJobSize(60);
+    $ecsConfig->parallelMaxNumberOfProcesses(16);
+    $ecsConfig->parallelTimeoutInSeconds(120);
```

Comment:
Probably leftover that needs an update now :)

Reply:
Thanks. Fixed.